sdk/instrumentation/net/http/contenttype.go

```diff
@@ -10,8 +11,14 @@ import (
 // - application/vnd.api+json
 // - application/x-www-form-urlencoded'
 var contentTypeAllowListLowerCase = []string{
-	"json",
-	"x-www-form-urlencoded",
+	"json", "x-www-form-urlencoded",
+}
+
+func SetContentTypeAllowList(config *agentconfig.AgentConfig) {
```

Comment:
Do we need this call and `contentTypeAllowListLowerCase` global state? Can we just get it from the config? Seems like we have a `GetConfig()` API available.

Reply:
+1 if it's easy to change from the hardcoded content types.